docs/markdown/markseries.md

```diff
@@ -112,3 +112,15 @@ Type: `function(info)`
 
 #### animation (optional)  
 See the [XYPlot](xy-plot.md)'s `animation` section for more information.
+
+## MarkSeriesGL API Additions
+
+In addition to the above api the GL version of markSeries exposes several additional props.
+
+#### seriesId
+Type: `string` (REQUIRED)
```

Comment:
maybe put the requireds next to the prop name to match how optionals are displayed too?